### DISCUSSION TITLE
add QR Code reader to enable TOTP

### DISCUSSION THREAD

**Discussion by seelenreiter** [keepassxreboot/keepassxc#6320] [Ideas]
I would be happy if you create the opportunity at the TOTP facility to scan the QR code directly or to read a picture.
Since I have made a "backup" of the QR codes and have reached the memory limit with the Yubikey, I would like to add the TOTP, which is not used regularly, to the KeepassXC database. Unfortunately, I now have to use an external tool to insert the secret. 
Best regards
SR

**Comment by droidmonkey** [comment]
#4825

**Reply by seelenreiter** [reply]
my fault.
